pages/release-notes.mdx

```diff
@@ -115,6 +115,30 @@ Memgraph (v2.11.1 and lower).
 
 </Callout>
 
+## Lab v2.13.0 - Apr 10, 2024
+
+{<h3> New features and improvements </h3>}
+
+- Select a color for your connections, enhancing distinction within
+  the Lab web interface.
+- Use precise CSV imports by defining custom line filters,
+  and processing specific file lines based on user-defined conditions.
+- Filter datasets by category such as "Knowledge Graphs", "Supply Chain",
+  and others.
+- Run the Cypher query from the query parameters tab without switching
+  back to the Cypher query tab in the query editor.
+- Enhanced CSV import now provides extensive notices, best practices
+  and warnings tailored to the CSV import process and specific data
+  types such as dates and date times, aiding you in smoother data handling.
+
+{<h3> Bug fixes </h3>}
+
+- Queries in a collection from a European backpacking dataset now return
+  a correct response.
+- Running a query in a horizontal view now works as expected.
+- A bug introduced in v2.12.0 with failed key encryption when
+  adding OpenAI and Azure OpenAI keys is fixed now.
+
```

Comment:
@Ana-Marija-design when you get a change can you check if this is ok for the release notes.

Reply:
Only wonder if we should emphasize "Filter **template** datasets"... Though I guess it's intuitive. Made me think a bit, though.

Looks good otherwise :)

Reply:
That's what I wrote the first time, but then I saw and we are not using "template" anywhere in the Lab, we just call it a dataset, so that's why I removed template from the word.

We even don't have "Template dataset" on the dataset cards now, category is now there.